mac

```diff
@@ -159,27 +158,48 @@ fancy_echo "Update heroku binary..."
 brew unlink heroku
 brew link --force heroku
 
-fancy_echo "Configuring Ruby ..."
-find_latest_ruby() {
-  rbenv install -l | grep -v - | tail -1 | sed -e 's/^ *//'
+fancy_echo "Configuring asdf version manager..."
+if [ !  -d "$HOME/.asdf" ]; then
+  git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.3.0
+  append_to_zshrc "source $HOME/.asdf/asdf.sh" 1
```

Comment:
I don't really fully understand that approach so I'd love to leave that aspect to someone else. 

My use case here is "run laptop" followed by "run rcm" which is going to wipe the zshrc here anyway.